app/controllers/users_controller.rb

```diff
@@ -0,0 +1,66 @@
+class UsersController < AuthenticationController
+  skip_before_action :authenticate, only: :create
+  # valid user creation curl --
+  #
+  # curl -X POST -F user[first_name]=Nate \
+  # -F user[state]=MD -F user[country]=USA \
+  # -F user[city]=Annapolis -F user[email]=test@test.com \
+  # -F user[last_name]=Bates  -F user[password]=Helloworld \
+  # http://localhost:3000/users.json -H "Accept: application/json"
+
+  # curl -i -H "Content-Type: application/json" \
+  # -d '{"user": {"email": "test@test.com"}}'
+
+  # POST /users
+  def create
+    result = CreateUser.call(user_params: user_params)
+    if result.success?
+      token = CreateAuthToken.call(user: result.user).token
+      render json: { jwt: token }, status: :created
+    else
+      render json: result.user.errors, status: :unprocessable_entity
+    end
+  end
+
+  # GET /users/1
+  def show
+    result = ShowUser.call(id: params[:id])
+    if result.success?
+      render json: result.user, status: :ok
+    else
+      render json: result.error, status: :not_found
+    end
+  end
```

Comment:
@Enriikke on `#show` and `#destroy` I've got `else` branches that handle errors. I don't often see this in other controllers I've looked at. Should I simplify these to the basic assignment (`@user = User.find(:id)` or similar) and forget about the `else`s? Seems like it's always better to have a safety net?